apps/web/package.json

```diff
@@ -6,21 +6,27 @@
   "license": "Apache-2.0",
   "author": "Hylo <hello@hylo.com>",
   "type": "module",
+  "engines": {
+    "node": "^20",
+    "yarn": "^4.5.0"
+  },
+  "packageManager": "yarn@4.5.0",
   "browserslist": [
     "defaults"
   ],
   "proxy": "localhost:3001",
   "scripts": {
     "viteDev": "vite serve",
     "dev": "yarn start",
-    "start": "rm -rf ../../node_modules/.cache/babel-loader && node --preserve-symlinks scripts/start.js",
```

Comment:
`rm -rf ../../node_modules/.cache/babel-loader` was needed in the past due to a bug in graqphql parsing which meant that changes to existing files would not be reflected even after a restart. Do you know if this cleared-up?

Reply:
i dont, but i will check

Reply:
it would be nice if we didn't need to do that, but as of last I checked it was still an issue. hopefully with the babel upgrades and such it isn't an issue anymore. to test make a change to an existing graphql query (probably one wrapped in `gql` or an actual `.graphql` file if we still have any ) and if the changes come through either on hot reload or restart without this extra bit, then we're golden... you probably already knew/remembered this, but just to make sure we're talking about the same thing :)